### PR TITLE
Feature/web stories block styles

### LINKED ISSUE #38: Rounded Corners Option
Add ~the option and style~ toggle button to have round corners for stories limited to the below view layouts. 

- [x] Grid view
- [x] List view
- [x] Carousel view

User should be able to choose between sharp and rounded corners.

### PR DESCRIPTION
**Pull request by divyarajmasani**
## Summary

- Replaces 'Show Cover image' control with 'Show Story Player' control to embed stories in-place.
- Adds new control to make story cards have rounded corners.

## Relevant Technical Choices

- By default, the blocks will show cover images.
- The control to toggle the corner style will be available only when cards are showing the cover image.
 
## To-do

- Maybe come up with a different name for the 'Replace cover image with story player' control.

## Testing Instructions

- Select 'Latest Web Stories' or 'Selected Block Stories' block in Editor.
- Observe new controls on the righthand sidebar.
- Toggle 'Replace Cover image with story player' to embed stories in-place.

Screenshot 2020-11-23 at 4 43 48 PM

- Toggle 'Show square corners' to make card corners rounded.

---

Fixes #38

### CODE REVIEW

File: assets/src/latest-stories-block/block/edit.css [hunk 1]
```diff
@@ -273,12 +265,19 @@
...
-  .latest-stories__inner-wrapper {
-  border: 1px solid #8080805e;
+/* Stories Block Styles */
+
+.wp-block-web-stories-latest-stories.is-style-default.is-view-type-grid
+  .latest-stories__controller,
+.wp-block-web-stories-latest-stories.is-style-default.is-view-type-list
+  .latest-stories__story-placeholder,
+.wp-block-web-stories-latest-stories.is-style-default.is-view-type-carousel
+  .latest-stories__story-wrapper {
```
Comment (RahiDroid) [inline assets/src/latest-stories-block/block/edit.css:278]: Can we not use `:not` operator to exclude circles view?

File: assets/src/latest-stories-block/block/edit.js [hunk 2]
```diff
@@ -144,7 +145,27 @@ const LatestStoriesEdit = ({ attributes, setAttributes }) => {
...
+        isShowingTitle: true,
+      });
+    }
+  }, [viewType, setAttributes]);
+
+  const willShowStoryPlayer =
+    'grid' !== viewType ? false : isShowingStoryPlayer;
+
   const willShowDate = 'circles' === viewType ? false : isShowingDate;
   const willShowAuthor = 'circles' === viewType ? false : isShowingAuthor;
```
Comment (RahiDroid) [inline assets/src/latest-stories-block/block/edit.js:170]: With this logic in place, I don't think we need these checks while rendering as they will always be based on the view type, did I understand it right? In short, can these checks `willShowDate`, `willShowAuthor`, etc can not be removed now? Also, should this useEffect not be moved to `latestStoriesControls.js` file as that's where these flags are being handled?

Comment (divyarajmasani) [reply assets/src/latest-stories-block/block/edit.js:170]: True, these checks are based on the view then we don't need them as we always set/update the state on view change.

Also, I feel keeping the state control in the main file is much easier to understand the block state logic.

File: assets/src/latest-stories-block/block/edit.css (hunk 1 above)
Comment (divyarajmasani) [reply assets/src/latest-stories-block/block/edit.css:278]: These are three different selectors, if it were to be the same then using `:not()` would've been helpful.

Comment (RahiDroid) [reply assets/src/latest-stories-block/block/edit.css:278]: My bad, didn't notice that last selectors are different 👍

**Review by RahiDroid — APPROVED** [review]

**Pull request merged**